2021-10-12 22:59:31,556 INFO    MainThread:15907 [wandb_setup.py:_flush():69] setting env: {}
2021-10-12 22:59:31,557 INFO    MainThread:15907 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-12 22:59:31,557 INFO    MainThread:15907 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-12 22:59:31,557 INFO    MainThread:15907 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-12 22:59:31,557 INFO    MainThread:15907 [wandb_init.py:_log_setup():342] Logging user logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211012_225931-3dkytbfi/logs/debug.log
2021-10-12 22:59:31,557 INFO    MainThread:15907 [wandb_init.py:_log_setup():343] Logging internal logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211012_225931-3dkytbfi/logs/debug-internal.log
2021-10-12 22:59:31,557 INFO    MainThread:15907 [wandb_init.py:init():375] calling init triggers
2021-10-12 22:59:31,557 INFO    MainThread:15907 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/bio_dataset/bio_ner_dataset.csv', 'checkpoint_path': '/root/konst/model_checkpoints/ner_employment', 'model_path': 'ner_base_epoch_11_acc_0.5074_vs_0.0804_loss_0.2233_vs_0.2597.pth', 'batch_size': 128, 'epochs': 160, 'start_epoch': 0, 'test_size': 0.05, 'lam': 0, 'patience': 5, 'factor': 0.5, 'use_wandb': True, 'lr': 0.00015, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 4}
2021-10-12 22:59:31,557 INFO    MainThread:15907 [wandb_init.py:init():424] starting backend
2021-10-12 22:59:31,557 INFO    MainThread:15907 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-12 22:59:31,562 INFO    MainThread:15907 [backend.py:ensure_launched():135] starting backend process...
2021-10-12 22:59:31,566 INFO    MainThread:15907 [backend.py:ensure_launched():140] started backend process with pid: 15932
2021-10-12 22:59:31,567 INFO    MainThread:15907 [wandb_init.py:init():429] backend started and connected
2021-10-12 22:59:31,569 INFO    MainThread:15907 [wandb_init.py:init():477] updated telemetry
2021-10-12 22:59:31,569 INFO    MainThread:15907 [wandb_init.py:init():500] communicating current version
2021-10-12 22:59:32,054 INFO    MainThread:15907 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-12 22:59:32,054 INFO    MainThread:15907 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-12 22:59:32,287 INFO    MainThread:15907 [wandb_init.py:init():540] starting run threads in backend
2021-10-12 22:59:33,047 INFO    MainThread:15907 [wandb_run.py:_console_start():1584] atexit reg
2021-10-12 22:59:33,047 INFO    MainThread:15907 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-10-12 22:59:33,048 INFO    MainThread:15907 [wandb_run.py:_redirect():1463] Redirecting console.
2021-10-12 22:59:33,049 INFO    MainThread:15907 [wandb_run.py:_redirect():1519] Redirects installed.
2021-10-12 22:59:33,049 INFO    MainThread:15907 [wandb_init.py:init():565] run started, returning control to user process
2021-10-12 23:44:11,891 INFO    MainThread:15907 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 255
2021-10-12 23:44:11,892 INFO    MainThread:15907 [wandb_run.py:_restore():1526] restore
